datasets/Van_de_Schoot_PTSD/convert_clean_schoot_lgmm_ptsd.ipynb

```diff
@@ -0,0 +1,282 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import pandas as pd\n",
+    "from risparser import convert_ris_to_csv\n",
+    "os.chdir(\"/automated-systematic-review-datasets/datasets/Van_de_Schoot_PTSD\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Convert RIS files into CSV"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Input file: raw/schoot-lgmm-ptsd-initial.ris\n",
+      "Number of articles: 6187\n",
+      "Export file: csv/schoot-lgmm-ptsd-initial.csv\n",
+      "Input file: raw/schoot-lgmm-ptsd-included-1.ris\n",
+      "Number of articles: 363\n",
+      "Export file: csv/schoot-lgmm-ptsd-included-1.csv\n",
+      "Input file: raw/schoot-lgmm-ptsd-included-3.ris\n",
+      "Number of articles: 8\n",
+      "Export file: csv/schoot-lgmm-ptsd-included-3.csv\n",
+      "Input file: raw/schoot-lgmm-ptsd-included-2.ris\n",
+      "Number of articles: 38\n",
+      "Export file: csv/schoot-lgmm-ptsd-included-2.csv\n"
+     ]
+    }
+   ],
+   "source": [
+    "raw_files = [raw_files for raw_files in os.listdir(\"raw\") if raw_files.endswith(\".ris\")]\n",
+    "os.makedirs(\"csv\", exist_ok=True)\n",
+    "for ris_file in raw_files :\n",
+    "    ris_fp = os.path.join(\"raw\", ris_file)\n",
+    "    csv_fp = os.path.join(\"csv\", os.path.splitext(ris_file)[0]+\".csv\")\n",
+    "    convert_ris_to_csv(ris_fp, csv_fp)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Clean the data \n",
+    "\n",
+    "(Code from the R file)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# original papers\n",
+    "os.chdir(\"csv\")\n",
+    "all_data = pd.read_csv(\"schoot-lgmm-ptsd-initial.csv\")\n",
+    "\n",
+    "# included papers\n",
+    "inc_data = pd.read_csv(\"schoot-lgmm-ptsd-included-2.csv\")\n",
+    "\n",
+    "# After abstract screening\n",
+    "aas_data = pd.read_csv(\"schoot-lgmm-ptsd-included-1.csv\")\n",
+    "\n",
+    "# Directly included after reading the abstract\n",
+    "dir_data = pd.read_csv(\"schoot-lgmm-ptsd-included-3.csv\")\n",
+    "\n",
```

Comment:
I think that instead of changing the working directory, it is more intuitive to indicate directory structure directly when you write a file

So you could get rid of `os.chdir("csv")`
and add the directory "csv\" to the filenames, like:
` all_data = pd.read_csv("csv/schoot-lgmm-ptsd-initial.csv"),`